tests/tx/test_verification.py

```diff
@@ -175,6 +176,40 @@ def test_block_verify(self) -> None:
         verify_parents_wrapped.assert_called_once()
         verify_height_wrapped.assert_called_once()
 
+    def test_block_validate_full(self) -> None:
```

Comment:
It's missing `verify_weight()` and `verify_reward()`. They are both called from `verify_basic()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/824/commits/408b1d147c6e83216782bd3311bfa599604c6508. I've also added specific `validate_basic()` tests.

Reply:
Well done. Thanks!